content/800-data-platform/400-platform-console/20-concepts/10-projects.mdx

```diff
@@ -0,0 +1,237 @@
+---
```

Comment:
For the Projects sections I want to keep;

# Projects
## API Keys

(with the below copy suggestions)

The rest can be removed.